### DISCUSSION TITLE
Tracking edits

### DISCUSSION THREAD

**Discussion by janetchapman** [organicmaps/organicmaps#2295] [Ideas]
When we used Maps.Me with Crowd2Map we could track the number of edits each month in Tanzania which was extremely useful. Is it possible to do this with Organic Maps please?

**Comment by biodranik** [comment]
Please contact Crowd2Map directly, they can extract statistics from publicly available OpenStreetMap changelogs.

**Comment by janetchapman** [comment]
Hi Alexander

I am Crowd2Map!

we used to have a tool for seeing a summary of maps.me contributions per month per country per user https://mmwatch.itinero.tech/?filters=1&date=06.2021&country=Uganda&nolimit=1

Is there any tool like this for Organic Maps?

Is very useful when organizing a campaign to contribute POIs to OSM with a group of people and for mapping parties, and the OSM community generally would welcome it.  Geoffrey also posted this in the Telegram group.

Thanks
Janet

**Reply by biodranik** [reply]
Please check this page https://wiki.openstreetmap.org/wiki/Editor_usage_stats

**Reply by kateregga1** [reply]
@biodranik what Janet was referring to was something like a dashboard to summarize contributions per user per country per month.
That would be useful for coordinating a mapping campaign for POIs

Something like this
![image](https://user-images.githubusercontent.com/1929397/160341271-b1cbe854-1b01-4965-953f-d6c7f390b824.png)

**Reply by biodranik** [reply]
I understand. There is a link to Zverik's script used to collect edit stats on the page above. It can be a good start to create a page like this.
